txn-recovery-marker-jdbc-hibernate5/module.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module xmlns="urn:jboss:module:1.5" name="io.narayana.openshift-recovery">
```

Comment:
perhaps this file should be in src/main/resources/modues/io/narayana/openshift-recovery/?

Reply:
ok, probably it should be